src/Nox.Types/Nox.Types.csproj

```diff
@@ -24,6 +24,7 @@
   <ItemGroup>
     <None Include="..\..\images\nox.png" Pack="true" PackagePath="" />
     <InternalsVisibleTo Include="Nox.Types.Tests" />
+    <InternalsVisibleTo Include="Nox.EntityFramework.SqlServer" />
```

Comment:
@dejobratic this should not be needed, prefer to make dependencies public

Reply:
Changed. Thanks.